docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,269 @@
+# Routes with the GraphQL Storefront API
+
+The BigCommerce GraphQL Storefront API allows you to retrieve data at the destination of a route (URL) for each storefront channel using the `route` node. This option makes it easier to build headless storefront applications, removing the need to track entity ID and type when accessing data for entities such as Products, Categories, Pages, Blogs, and more.
+
+The destination URL is the only required input when using the `route` node. You will provide an input with the `path` parameter on the `route` node. The response will be an entity at that URL, a redirect to a new location, or nothing if the URL does not relate to an entity or redirect.
+
+When querying a `route`, the response may include the following node types:
+
+* Product
+* Category
+* Brand
+* NormalPage
+* ContactPage
+* RawHtmlPage
+* Blog
+* BlogPost
+
+## Redirect support
+
+The `route` node supports 301 Redirects that you have added to the store. If the URL provided as input to the `route`, references a redirect, then the response will return a `Redirect` node, which comes in two varieties: a manual redirect or a dynamic redirect.
+
+A `ManualRedirect` returns a fixed destination URL. You will need a new `route` query to attempt to pull the content at that destination URL. If the destination URL itself is a redirect, this can create a chain of redirects requiring multiple queries.
+
+A "dynamic redirect" returns an entity type and entity ID of an existing store resource. There are five types of dynamic redirects: `ProductRedirect`, `BrandRedirect`, `BlogPostRedirect`, `CategoryRedirect`, and `PageRedirect`. Once the entity type and ID are retrieved, you can initiate a new query to pull that content from the appropriate resource node. With dynamic redirects, you can retrieve the target entity directly within the `route` query response, avoiding an additional request. We will detail how this works further below.
```

Comment:
I'm not liking the use of "We" here since I don't think we are writing with this same perspective in other places. Maybe phrase this line:
> We will detail how this works further below.

as:
> This is detailed below.